app/models/collection.rb

```diff
@@ -321,28 +315,44 @@ def gift_notification
     self.collection_profile.gift_notification || (parent ? parent.collection_profile.gift_notification : "")
   end
 
-  def moderated? ; self.collection_preference.moderated ; end
-  def closed? ; self.collection_preference.closed ; end
-  def unrevealed? ; self.collection_preference.unrevealed ; end
-  def anonymous? ; self.collection_preference.anonymous ; end
-  def challenge? ; !self.challenge.nil? ; end
+  def moderated?() = self.collection_preference.moderated
+
+  def closed?() = self.collection_preference.closed
+
+  def unrevealed?() = self.collection_preference.unrevealed
+
+  def anonymous?() = self.collection_preference.anonymous
+
+  def challenge?() = !self.challenge.nil?
 
   def gift_exchange?
-    return self.challenge_type == "GiftExchange"
+    self.challenge_type == "GiftExchange"
   end
+
   def prompt_meme?
-    return self.challenge_type == "PromptMeme"
+    self.challenge_type == "PromptMeme"
   end
 
-  def get_maintainers_email
-    return self.email if !self.email.blank?
-    return parent.email if parent && !parent.email.blank?
-    "#{self.maintainers.collect(&:user).flatten.uniq.collect(&:email).join(',')}"
+  def maintainers_list
+    self.maintainers.collect(&:user).flatten.uniq
+  end
+
+  def notify_maintainers_challenge_sent
+    # loop through maintainers and send each a notice via email
+    self.maintainers_list.each do |user|
+      I18n.with_locale(user.preference.locale.iso) do
+        UserMailer.collection_notification(self.id, I18n.t("user_mailer.collection_notification.assignments_sent.subject"), I18n.t("user_mailer.collection_notification.assignments_sent.complete"), user.email).deliver_later
```

Comment:
Would be nice if you could pull the subject and message out into variables so this method call doesn't get so long.